tools/jarcat/zip/writer.go

```diff
@@ -338,8 +338,20 @@ func (f *File) AddInitPyFiles() error {
 }
 
 // AddManifest adds a manifest to the given zip writer with a Main-Class entry (and a couple of others)
-func (f *File) AddManifest(mainClass string) error {
-	manifest := fmt.Sprintf("Manifest-Version: 1.0\nMain-Class: %s\n", mainClass)
+func (f *File) AddManifest(mainClass string, classPath string) error {
+
+	if mainClass == "" && classPath == "" {
```

Comment:
not sure about this. I think this function should always write the manifest, the decision of whether to do it or not should be done by the caller.